luminance/src/tess.rs

```diff
@@ -393,24 +399,54 @@ pub enum TessError {
   Overflow(usize, usize)
 }
 
-pub struct Tess {
+pub unsafe trait VertIndex {
```

Comment:
Maybe we can simplify that trait a lot by introducing only an associated value via a constant, like it’s done with `VertexAttribFmt`:

```rust
trait Index {
  const INDEX_FMT: IndexFmt;
}

// …

enum IndexFmt {
  U8,
  U16,
  U32
}
```

Reply:
Then, in the implementation, we can simply have a function that maps `IndexFmt` to the required set of OpenGL attributes. That is going to be needed to enable backends (other versions of GL, WebGL, GL ES, maybe Vulkan, etc.)

Reply:
Also, I think we can name that `TessIndex`, as those are indexes used to build up tessellations.